ui/lib/core/addon/components/search-select-with-modal.hbs

```diff
@@ -0,0 +1,107 @@
+<div {{did-insert this.fetchOptions}} ...attributes data-test-search-select-with-modal>
+  {{#if this.shouldUseFallback}}
+    {{component
+      @fallbackComponent
+      label=(or @label @subLabel)
+      subText=@subText
+      onChange=@onChange
+      inputValue=@inputValue
+      helpText=@helpText
+      placeHolder=@placeHolder
+      id=@id
+    }}
+  {{else}}
+    {{#if @label}}
+      <label for={{or @id @label}} class="is-label" data-test-field-label>
+        {{capitalize @label}}
+        name
+        {{#if @helpText}}
+          <InfoTooltip>{{@helpText}}</InfoTooltip>
+        {{/if}}
+      </label>
+    {{/if}}
+    {{#if @subLabel}}
```

Comment:
I wonder if we need sublabel? It's not in the class attributes and we can add it in later if we need it

Reply:
good call!